doc/js_tutorials/js_assets/js_image_classification.html

```diff
@@ -0,0 +1,384 @@
+    <textarea class="code" rows="9" cols="100" id="codeEditor2" spellcheck="false"></textarea>
+    <p>4.Get blob from image as input for net, and standardize it with <b>mean</b> and <b>std</b>.</p>
+    <textarea class="code" rows="11" cols="100" id="codeEditor3" spellcheck="false"></textarea>
+    <p>5.Fetch model from network and save to emscripten file system.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor4" spellcheck="false"></textarea>
+    <p>6.The post-processing, including softmax if needed and get the top classes from the output vector.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor5" spellcheck="false"></textarea>
+    <p class="err" id="errorMessage"></p>
+</div>
+
+<div id="appendix">
+    <h2>Model Info:</h2>
+</div>
+
+<script src="utils.js" type="text/javascript"></script>
+
+<script id="codeSnippet" type="text/code-snippet">
+inputSize = [224,224];
+mean = [104, 117, 123];
+std = 1;
+rgb = true;
+
+// record if need softmax function for post-processing
+needSoftmax = false; 
+
+// url for model, config and label files, can from native or Internet
+// for model type like onnx which doesn't need config file, just let configUrl = ""
+labelsUrl = "labels1000.txt";
+modelUrl = "bvlc_googlenet.caffemodel";
+configUrl = "bvlc_googlenet.prototxt";
+</script>
+
+<script id="codeSnippet1" type="text/code-snippet">
+main = async function() {
+    const url = {
+        modelUrl: modelUrl,
+        configUrl: configUrl
+    }; 
+    const path = await loadModel(url);
+    const labels = await loadLables(labelsUrl);
+    const input = getBlodFromImage(inputSize, mean, std, rgb);
+    let net;
+    if(path.configPath == "") {
+        net = cv.readNet(path.modelPath);
+    } else {
+        net = cv.readNet(path.configPath, path.modelPath);
+    }
+    net.setInput(input);
+    const start = performance.now();
+    const result = net.forward();
+    const time  = performance.now()-start;
+    const probs = softmax(result);
+    const classes = getTopClasses(probs, labels);
+
+    updateResult(classes, time);
+    input.delete();
+    net.delete();
+    result.delete();
+}
+</script>
+
+<script id="codeSnippet2" type="text/code-snippet">
+loadLables = async function(labelsUrl){
+    document.getElementById('status').innerHTML = 'Loading classification labels...';
+    let response = await fetch(labelsUrl);
+    let label = await response.text();
+    label = label.split('\n');
+    document.getElementById('status').innerHTML = '';
+    return label;
+}
+</script>
+
+<script id="codeSnippet3" type="text/code-snippet">
+getBlodFromImage = function() {
+    let mat = cv.imread("canvasInput");
+    let matC3 = new cv.Mat(mat.matSize[0],mat.matSize[1],cv.CV_8UC3);
+    cv.cvtColor(mat, matC3, cv.COLOR_RGBA2RGB);
+    let input = cv.blobFromImage(matC3, std, new cv.Size(inputSize[0], inputSize[1]), 
+                                 new cv.Scalar(mean[0], mean[1], mean[2]), rgb);
```

Comment:
Do all the models process RGB images or BGR?

Reply:
Not really, some models need to change the channel. So I set a global flag to control it.

Reply:
Sorry, just missed that you use `rgb` flag already.

Reply:
BTW, seems like you need `!rgb` because
`image (RGBA) -> cvtColor RGB -> blob`
and this flag swaps R and B (for RGB) or B and R (for BGR). So to keep RGB input unchanged, you need `!rgb`

Reply:
That make sense. `rgb` comes from the input argument in the python example. Maybe change the name of this flag to `swapRB` is more suitable?

Reply:
Not sure. Let's keep the same config (RGB flag) but just manage it in a right way in code.